pkg/registry/openshift_ca.go

```diff
@@ -0,0 +1,87 @@
+package registry
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/openshift/special-resource-operator/pkg/clients"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+type openShiftCAGetter struct {
+	kubeClient clients.ClientsInterface
+}
+
+//go:generate mockgen -source=openshift_ca.go -package=registry -destination=mock_openshift_ca_api.go
+
+type OpenShiftCAGetter interface {
+	AdditionalTrustedCAs(ctx context.Context) (map[string][]byte, error)
+	CABundle(context.Context) ([]byte, error)
```

Comment:
```suggestion
	CABundle(ctx context.Context) ([]byte, error)
```
Just for consistency :)

Reply:
Done.